frontend/packages/knative-plugin/src/components/overview/DeploymentOverviewList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { PodControllerOverviewItem } from '@console/shared';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import { ResourceLink } from '@console/internal/components/utils';
+
+export type DeploymentOverviewListProps = {
+  current: PodControllerOverviewItem;
+};
+
+const DeploymentOverviewList: React.FC<DeploymentOverviewListProps> = ({ current }) => {
```

Comment:
Destructure the obj here itself.
```suggestion
const DeploymentOverviewList: React.FC<DeploymentOverviewListProps> = ({ current: { obj } }) => {
```